owncloudApp/src/test/java/com/owncloud/android/presentation/sharing/PublicLinkPasswordUtilsTest.kt

```diff
@@ -0,0 +1,102 @@
+/**
+ * ownCloud Android client application
+ *
+ * @author Juan Carlos Garrote Gascón
+ *
+ * Copyright (C) 2024 ownCloud GmbH.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.owncloud.android.presentation.sharing
+
+import org.junit.Assert.assertTrue
+import org.junit.Test
+
+class PublicLinkPasswordUtilsTest {
+
+    private val charsetLowercase = ('a'..'z').toList()
+    private val charsetUppercase = ('A'..'Z').toList()
+    private val charsetDigits = ('0'..'9').toList()
+    private val charsetSpecial = listOf('!', '#', '$', '%', '&', '\'', '(', ')', '*', '+', ',', '-', '.', '/', ':', ';', '<', '=', '>', '?', '@', '[', '\\', ']', '^', '_', '`', '{', '|', '}', '~')
+
+    private val minCharacters = 10
+    private val maxCharacters = 100
+    private val minDigits = 2
+    private val minLowercaseCharacters = 2
+    private val minUppercaseCharacters = 2
+    private val minSpecialCharacters = 2
```

Comment:
why don't we put the values that are coming right now from the server?

Reply:
We're doing a isolation unit test, how are we going to know which values are coming from the server? That doesn't make sense